src/status_im/contexts/profile/login/events.cljs

```diff
@@ -104,7 +103,7 @@
            [:chat.ui/request-link-preview-whitelist]
            [:visibility-status-updates/fetch]
            [:switcher-cards/fetch]
-           (when (ff/enabled? ::ff/wallet.wallet-connect)
+           (when true
```

Comment:
I think you missed this (and other `ff` changes) when pushing. 

Also, if it helps, for development I just added a `.env.debug` file with `FLAG_WALLET_CONNECT_ENABLED=1`, so I don't have to always change it in the code (it would be very annoying for me to do this always).